src/app/docs/search/search-docs.component.html

```diff
@@ -4,8 +4,58 @@
   pageTitle="Search"
 >
 
-<sky-docs-demo-page-summary>
-  The search component creates an input control for users to enter search criteria.
-</sky-docs-demo-page-summary>
+  <sky-docs-demo-page-summary>
+    The search component creates an input control for users to enter search criteria.
+  </sky-docs-demo-page-summary>
 
-</sky-docs-demo-page>
\ No newline at end of file
+  <sky-docs-demo>
+    <sky-toolbar>
+      <sky-toolbar-item>
+        <button
+          class="sky-btn sky-btn-default sky-demo-search-bar-item"
+          (click)="searchApplied('Robert')"
+        >
+          Predefined search text
+        </button>
+      </sky-toolbar-item>
+      <sky-toolbar-item>
+        <sky-search
+          [searchText]="searchText"
+          [debounceTime]="250"
+          (searchApply)="searchApplied($event)"
+          (searchChange)="searchApplied($event)"
+        >
+        </sky-search>
+      </sky-toolbar-item>
+    </sky-toolbar>
+    <sky-repeater
+      expandMode="none"
```

Comment:
Should we set `expandMode` to `multiple` now that the demo is at the top of the page? As is, I don't see anything but the demo when the page loads. If we collapse the repeaters when they load, I think it would be a better use of space.

Reply:
Not sure... does that have an impact to the UX? @Blackbaud-ToddRoberts would it be OK to show the search demo with collapsed repeater results?

Reply:
would rather remove the description text from the repeater rows than put expand mode on them if we are worried about height

Reply:
Sounds good. I removed the description text.